bootstrap/manifest_template.json

```diff
@@ -67,6 +67,34 @@
     "url": "http://postgres-api.discoverd/ping"
   },
   {
+    "id": "logaggregator",
```

Comment:
@titanous I think you asked me to put this component after all of the controller stuff (i.e. after the last controller step, `scheduler`), but @benburkert wouldn't have known that. Is the current ordering fine or do you want it after `scheduler`?